client/lib/source/Upstart.cpp

```diff
@@ -96,7 +96,7 @@ bool Upstart::invokeMethod(const std::string& method,
             (std::chrono::steady_clock::now() - startTime);
     if (timeTaken > std::chrono::seconds(2))
     {
-        AI_LOG_WARN("upstart request took a rather long time (%llums)",
+        AI_LOG_WARN("upstart request took a rather long time (%ldms)",
```

Comment:
I believe we should switch to PRId64 instead of %lld here. The %lld -> %ld change system dependent, and on 32 bit systems we'd get this same warning again but the other way around.

Refer to:

https://linux.m2osw.com/c-printf3-int-long-and-long-long-works-every-time

Reply:
Fixed